src/core/Microsoft.PowerToys.Settings.UI.Lib/ViewModels/FancyZonesViewModel.cs

```diff
@@ -524,5 +528,12 @@ public void RaisePropertyChanged([CallerMemberName] string propertyName = null)
                 SendConfigMSG(ipcMessage.ToJsonString());
             }
         }
+
+        private string ToRGBHex(string color)
+        {
+            int argb = int.Parse(color.Replace("#", string.Empty), System.Globalization.NumberStyles.HexNumber);
+            Color clr = Color.FromArgb(argb);
```

Comment:
Color.FromArgb can throw an exception?
If yes, let's put a try catch and default it to `#FFFFFF`

Reply:
`int.Parse` also throws, after that's fixed - LGTM.

Reply:
Done.